File: Leetcode/palindromeNumber/palindromeNumber.py
def palindromeNumber (x):
  numberStr = str(x);
  numberReverse = '';

  for i in range(len(numberStr) -1, -1, -1):
    numberReverse += numberStr[i];

  if(numberStr == numberReverse): return print('true');

  return print('false');

x = 121;
palindromeNumber(x);

File: Leetcode/palindromeNumber/palindromeNumber.js
const palindromeNumber = (x) =>{
  let numberStr = x.toString();
  let reverseNumberStr = '';

  //Metodo para inverter o inteiro x
  for(let i = numberStr.length - 1; i >= 0; i--){
    reverseNumberStr += numberStr[i];
  }

  if(numberStr == reverseNumberStr){
    return true;
  }

  return false;
}

const x = 10;
palindromeNumber(x);
